### ISSUE TITLE
multivariate comparaions - p value correction & post hoc parameter choice

### ISSUE THREAD

**Issue by liaoandi** [uchicago-computation-workshop/abdullah_almaatouq#30]
Hi, thanks for the presentation! It is interesting. 
However, I have a question about statistical analysis. 
As you used multiple groups, multiple conditions and multiple phases to compare, maybe it is essential to use p-value correction methods? 
Plus, when choosing the top-k participants, similar correction techniques might be necessary in post-hoc parameter choice.  
Thanks!

**Comment by amaatouq** [comment]
yes, all p-values on individual-level outcomes are based on White cluster-robust standard errors to account for correlated error terms within each group. Also, all analyses/hypothesis/comparisons were pre-registered.
